redis.stub.php

```diff
@@ -408,8 +408,7 @@ public function script(string $command, mixed ...$args): mixed;
 
     public function select(int $db): Redis|bool;
 
-    /** @return bool|Redis */
-    public function set(string $key, mixed $value, mixed $opt = NULL);
+    public function set(string $key, mixed $value, ?mixed $opt = NULL): Redis|string|bool;
```

Comment:
?mixed => mixed

Reply:
Is there a formal definition for the difference between `?mixed` and `mixed` when the argument has a default value of NULL?